tools/test.py

```diff
@@ -1265,6 +1265,9 @@ def BuildOptions():
   result.add_option("--no-store-unexpected-output", 
       help="Deletes the temporary JS files from tests that fails",
       dest="store_unexpected_output", action="store_false")
+  result.add_option("-r", "--run",
+      help="Run every mth test starting from n (--run=n,m with n < m)",
```

Comment:
Or even better, not to focus too much on implementation details, we could say: `Divide the tests in m groups and run tests from group n`. Is that clearer? That's really the objective. The modulo is an implementation detail.